boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpunet.dts

```diff
@@ -23,6 +23,22 @@
 		zephyr,flash = &flash1;
 		zephyr,code-partition = &slot0_partition;
 	};
+
+	leds:	leds {
+		compatible = "gpio-leds";
+		rgb2_red: led_1 {
```

Comment:
Why not start at led_0?

Reply:
Because these adhere to what is stated on the board silkscreen.

Reply:
You may be referring to the other label. That could start from zero, yes.